From: Miroslav Benes <mbenes@suse.cz>
Subject: powerpc/livepatch: Fix KABI breaker in stacktrace.c
Patch-mainline: Never, KABI
References: bsc#1071995 bsc#1072856 bsc#1087458 bsc#1089664 bsc#1089669
 bsc#1090522 fate#323487

Several structures became defined after header files inclusion. Hide the
files from the checker.

Signed-off-by: Miroslav Benes <mbenes@suse.cz>
---
 arch/powerpc/kernel/stacktrace.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/powerpc/kernel/stacktrace.c
+++ b/arch/powerpc/kernel/stacktrace.c
@@ -20,7 +20,9 @@
 #include <asm/ptrace.h>
 #include <asm/processor.h>
+#ifndef __GENKSYMS__
 #include <linux/ftrace.h>
 #include <asm/kprobes.h>
+#endif
 
 /*
  * Save stack-backtrace addresses into a stack_trace buffer.
